symengine/series.h

```diff
@@ -349,14 +346,13 @@ class SeriesBase : public SeriesCoeffInterface
         }
     }
 
-    static inline Poly series_cot(const Poly &s, const Poly &var,
-                                  unsigned int prec)
+    static Poly series_cot(const Poly &s, const Poly &var, unsigned int prec)
```

Comment:
But isn't this defined inside class `SeriesBase`? And as far as I can recall templates are also automatically inlined. Am I missing something?